contrib/jenkins_tests/cov.sh

```diff
@@ -24,7 +24,7 @@ make clean >> "${cov_dir}/cov.log" 2>&1
 sleep 1
 eval "cov-configure --config ${cov_dir}/coverity_config.xml --gcc >> ${cov_dir}/cov.log 2>&1"
 sleep 1
-eval "cov-build --config ${cov_dir}/coverity_config.xml --dir ${cov_build} make -j >> ${cov_dir}/cov.log 2>&1"
+eval "cov-build --config ${cov_dir}/coverity_config.xml --dir ${cov_build} make -j8 >> ${cov_dir}/cov.log 2>&1"
```

Comment:
please consider using `make ${make_opt}`

Reply:
fixed

Reply:
@igor-ivanov , please review and approve